com.unity.netcode.gameobjects/Tests/Editor/com.unity.netcode.editortests.asmdef

```diff
@@ -9,10 +9,9 @@
         "Unity.Multiplayer.MetricTypes",
         "Unity.Multiplayer.NetStats",
         "Unity.Multiplayer.Tools.MetricTypes",
-        "Unity.Multiplayer.Tools.NetStats"
-    ],
-    "optionalUnityReferences": [
-        "TestAssemblies"
+        "Unity.Multiplayer.Tools.NetStats",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner"
```

Comment:
doesn't `TestAssemlies` cover `UnityEngine.TestRunner` & `UnityEditor.TestRunner` already? (in a safer way)

Reply:
Yeah, I was trying to keep it consistent with the rest of the asmdef files that do it this way.
Reverted that out and keeping the optionalUnityReferences

Reply:
Actually, if you use the previous way it causes issues in newer versions of unity.
The above is what is applied by default if you use the optionalUnityReferences... by including that it creates duplicate constraint entries for UNITY_INCLUDE_TESTS and it creates duplicate nunit.framework.dll assembly references.  When you open the asmdef in the editor, remove the duplicate entries, and then hit apply the above is what you get.